### ISSUE TITLE
Remove Milligram package

### ISSUE THREAD

**Issue by agonzalez0515** [gwg-women/gwg-women-techmakers#128]
In the early days of the app I added a package called Milligram, it's a lightweight UI framework. I was just using it for its super simple columns/row code while working on sidebar component. 

It's definitely not needed anymore and it's just adding extra weight to the app so it should be removed.  After it's removed, some of the styling will be affected and will need to be fixed.

**Comment by PepperAddict** [comment]
okay. I deleted milligrams in the package.json and the width duplicate. #119 
For future reference, having any 3rd party script from a place you don't trust is a security risk. 
While we're on this topic, are there any other packages that is no longer needed?

**Comment by khusbuchandra** [comment]
Removal of Milligram has some effect with the look of the app. So we decided to remove it from the PR and make those changes along with the fixes required to make the app look good post its removal. @PepperAddict , Would you like to take it up in fixing the issues that come up after removing it.
Image shared by @sonalikatara 
![image](https://user-images.githubusercontent.com/35678241/38130057-1f8b06da-341f-11e8-94a9-0af02bc9aa6f.png)

**Comment by PepperAddict** [comment]
Sure @khusbuchandra. Seems doable. 
@digilou I think you're working on a css task right now.  Is it okay if I take up this one to make it look milligramy or do you think I should wait till after #97 . Or if you want, you can take this one with #97 . It's really up to you and I don't mind doing this task and/or waiting. Just let me know. :) Thanks.

**Comment by digilou** [comment]
@PepperAddict Thanks for checking in. I can definitely look since I'm on that file now. Once I finish with #97 I'll tag you so you can take a look.
